### DISCUSSION TITLE
Authentication Error "The maximum number of teams has been reached"

### DISCUSSION THREAD

**Discussion by Pashaflayt** [outline/outline#3816] [Self Hosting] [answered]
I have the 0.65.1 version installed by Docker.
A configuration with one or two Auth providers causes an error: A configuration with one or two providers causes an error. Just redirect to https://mydomain/?notice=authentication-required and nothing happens.
In v0.64.4 auth flow through a single provider was successful.

**To Reproduce**
1. Set env for Google auth
2. Run docker-compose
3. Sign in with Google
4. Nothing happens

**Expected behavior**
Success auth flow

<details>
<summary> Some logs</summary>
{"error":"The maximum number of teams has been reached","stack":"UnauthorizedError: The maximum number of teams has been reached\n    at AuthenticationError (/opt/outline/build/server/errors.js:35:34)\n    at accountProvisioner (/opt/outline/build/server/commands/accountProvisioner.js:45:43)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Strategy._verify (/opt/outline/build/server/routes/auth/providers/google.js:66:18)","level":"error","message":"Error during authentication"}

{"error":"The maximum number of teams has been reached","stack":"UnauthorizedError: The maximum number of teams has been reached\n    at AuthenticationError (/opt/outline/build/server/errors.js:35:34)\n    at accountProvisioner (/opt/outline/build/server/commands/accountProvisioner.js:45:43)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async OAuth2Strategy._verify (/opt/outline/build/server/routes/auth/providers/oidc.js:83:22)","level":"error","message":"Error during authentication"}
</details>

**Outline (please complete the following information):**
- Install: self hosted
- Version: [4ee3929](https://github.com/outline/outline/commit/4ee3929e9dbfe29d48e8a1cc593fff8e4631f833)

**Desktop (please complete the following information):**
 - OS: MacOS 11.6.7
 - Browser Chrome
 - Version 103.0.5060.114 (arm64)

**Comment by tommoor** [comment]
One team is already configured, are you able to login with that team?

**Reply by Pashaflayt** [reply]
> One team is already configured, are you able to login with that team?

Can't login at all. How to reset team settings? Drop Postgres db and docker volumes doesn't help. I would appreciate any hint.

**Comment by TruongNguyen5395** [comment]
I'm having the same issue

**Reply by Eminlin** [reply]
+1

**Comment by MCMrARM** [comment]
Is this issue not caused by this change?
https://github.com/outline/outline/commit/28439d315dd9c7cf1681d6f2e1251da54a11beae#diff-c1cd0c3a0ab14d41ae176506fb7a755bb8ff1137945e488f31487224f3559608R78

I think it makes it impossible to create the initial team.

(I seem to have the same issue).

**Reply by MCMrARM** [reply]
I have commented out this line in question and was able to successfully create the initial team (before that I had zero teams).

**Reply by almereyda** [reply]
Good spot! The doubling of the outcome with the conditional above really seems misplaced here.

**Comment by tommoor** [comment, marked as answer]
This was fixed in **v0.65.2**
